### ISSUE TITLE
LiveBook

### ISSUE THREAD

**Issue by nelsonic** [dwyl/learn-phoenix-framework#147]
Programmer Passport: LiveBook - Video 1: LiveBook Scenarios: https://youtu.be/_Bj9Y0V8WgA

cc: @SimonLab

**Comment by SimonLab** [comment]
LiveBook collaborative feature: https://youtu.be/RKvqc-UEe34?t=1383

**Comment by nelsonic** [comment]
@SimonLab have you tried running it on your `localhost` ? 💭

**Comment by SimonLab** [comment]
Yes, running on Ubuntu was easy. Because I have already Erlang/Elixir setup I just ran the following commands:
https://github.com/livebook-dev/livebook#escript

```sh
mix escript.install hex livebook

# add livebook to path (I'm using asdf)
asdf reshim elixir

# Start the Livebook server
livebook server
```
result:
```
[Livebook] Application running at http://localhost:8080/?token=7qm2om3hmp75v2zxkwxlspopov44mhhv
```

![image](https://user-images.githubusercontent.com/6057298/162065598-0194dc30-56db-475b-8385-88058abf9343.png)

**Comment by nelsonic** [comment]
I did a similar install, but got the following warning along the way:

```sh
warning: you must append "/Users/n/.mix/escripts" to your PATH if you want to invoke escripts by name
```

Which meant that `livebook` was not installed on the path. 

```sh
zsh: command not found: livebook
```

Added `$HOME/.mix/escripts` to `PATH` in `~/.bashrc` and restarted terminal ...

After granting permissions:

<img width="372" alt="image" src="https://user-images.githubusercontent.com/194400/162073833-22d6c1f6-808c-46b3-97f2-f362e0f879f1.png">


I get a similar result:

<img width="569" alt="image" src="https://user-images.githubusercontent.com/194400/162074618-67748a4b-0b02-4ea3-81c1-43712fbee6f2.png">

What _next_? 💭
